<ui version="4.0" >
 <class>restoreTreeForm</class>
 <widget class="QWidget" name="restoreTreeForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>160</y>
     <width>382</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="jobTabelLabel" >
       <property name="text" >
        <string>Jobs</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="jobTable" />
     </item>
     <item>
      <widget class="QLabel" name="prLabel1" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="prBar1" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="value" >
        <number>24</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="prLabel2" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="prBar2" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="value" >
        <number>24</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="directoryTree" />
   <widget class="QWidget" name="layoutWidget_1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="fileLabel" >
       <property name="text" >
        <string>Files</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="fileTable" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="versionLabel" >
       <property name="text" >
        <string>Versions of File</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="versionFileLabel" >
       <property name="text" >
        <string>FileName</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="versionTable" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>57</y>
     <width>791</width>
     <height>79</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>950</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="refreshButton" >
        <property name="text" >
         <string>Refresh</string>
        </property>
        <property name="icon" >
         <iconset resource="../main.qrc" >:/images/view-refresh.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restoreButton" >
        <property name="text" >
         <string>Restore</string>
        </property>
        <property name="icon" >
         <iconset resource="../main.qrc" >:/images/restore.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="jobLabel" >
          <property name="text" >
           <string>Job</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="jobCombo" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Job List Job Criterion Selector</string>
          </property>
          <property name="statusTip" >
           <string>Job List Job Criterion Selector</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="clientLabel" >
          <property name="text" >
           <string>Client</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="clientCombo" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Job List Client Criterion Selector</string>
          </property>
          <property name="statusTip" >
           <string>Job List Client Criterion Selector</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="fileSetLabel" >
          <property name="text" >
           <string>FileSet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="fileSetCombo" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Job List Fileset Criterion Selector</string>
          </property>
          <property name="statusTip" >
           <string>Job List Fileset Criterion Selector</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="limitCheckBox" >
          <property name="text" >
           <string>Record Limit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="limitSpinBox" >
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="singleStep" >
           <number>25</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="daysCheckBox" >
          <property name="text" >
           <string>Days Limit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="daysSpinBox" >
          <property name="singleStep" >
           <number>7</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="directoryLabel" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" >
       <cursor>13</cursor>
      </property>
      <property name="text" >
       <string>Directory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSelectDirectory" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Select Directory</string>
   </property>
  </action>
  <action name="actionUnselectDirectory" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>UnselectDirectory</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
